submissions/Tale-Weaver

```diff
@@ -0,0 +1,42 @@
+# Tale Weaver
+
+### Project Summary
+> Tale Weaver stands out as an interactive storytelling marvel, leveraging AI to craft immersive narratives, beautifully rendered through DALL·E. Dive into a world where every choice shapes your journey, creating a dynamic narrative unlike any other.
+
+But what sets Tale Weaver apart isn't just its captivating tales; it's the integration of blockchain technology via Dojo. This ensures your adventures are not only thrilling but also secure, with every decision etched into the digital ledger for unparalleled transparency and composability.
+
+As we gear up to unleash Tale Weaver into the gaming sphere, we're not just aiming for production deployment. Oh no, we're taking it up a notch by integrating NFTs, specifically Bloberts, into the mix.
+
+Picture this: a contest mode where players pit their Bloberts against each other, battling for supremacy and bragging rights. Your Blobert isn't just an avatar; it's your digital alter ego, representing your skills and style in the Tale Weaver universe.
+
+While we've already nailed down the Blobert integration locally, the real challenge lies in ensuring its seamless transition to production for the game jam. But fear not, for every obstacle we encounter only fuels our dedication to push the boundaries of interactive storytelling, visual excellence, and digital ownership.
+
+Join us as we embark on this epic journey, where the fate of Bloberts hangs in the balance, and the potential of video games as platforms for innovation knows no bounds. Get ready to unleash your Blobert and prove its mettle in the ultimate contest of longevity!
+
+### GitHub
+taleweaver-ai
+
+### Play
+> Contact us for API keys and further instructions
```

Comment:
Please provide more details here

Reply:
We updated the README.md. Although we think it will be difficult to try it out based on the API keys needed, our goal was to have it deployed on Vercel for you to play it. We were willing to use our API key and its costs but we didn't really wanted to expose sensitive info and didn't have enough time to do it.